packages/astro/package.json

```diff
@@ -164,7 +156,6 @@
     "strip-ansi": "^7.0.1",
     "supports-esm": "^1.0.0",
     "tsconfig-resolver": "^3.0.1",
-    "typescript": "*",
```

Comment:
This is necessary for `astro check` to work, but it does bring up a good question of if Astro 2.0 is a good time to change this!

Reply:
Ah I couldn't find the peer dep for this. Thanks for pointing out, I'll revert this for now. I do like if Astro doesn't install TypeScript by default, but we could investigate that separately!

Reply:
Yep! Spun up a thread on Discord asking for people's opinion on this. We can tackle separately for sure!

Reply:
I've reverted this 👍